submission/launch.sh

```diff
@@ -24,6 +24,7 @@ RESULT_JSON=$(jq -n \
                  --arg out "$stdout" \
                  --arg err "$stderr" \
                  --arg code $exit_code \
-                 '{stdout: $out, stderr: $err, exit_code: $code,}')
+                 --arg auth_token "$VMCK_AUTH" \
+                 '{stdout: $out, stderr: $err, exit_code: $code, auth: $auth_token,}')
```

Comment:
Maybe it would be better to include the auth token in the url you're already passing along?

Reply:
Why would that be the case?

Reply:
Because then you don't need to pass around another value. You can include it inside the URL: `http://acs-interface/submission/<pk>/done?token=<token>`.